### DISCUSSION TITLE
Where is the documentation for generated types?

### DISCUSSION THREAD

**Discussion by KevinMusgrave** [prisma/prisma#17054] [Q&A]
There are some examples of generated types here: https://www.prisma.io/docs/concepts/components/prisma-client/advanced-type-safety#what-are-generated-types.

But I haven't been able to find a page that lists all generated types and their meanings.

**Comment by nurul3101** [comment]
Hello @KevinMusgrave 👋 

The generated types are dependent on your models, but I believe we can have a page which lists all the generated types based on a sample schema. Do you mind creating a [GitHub Docs Issue](https://github.com/prisma/docs/issues/new?assignees=&labels=docs&template=doc_request.md) in the prisma docs repository so that our docs team can provide their inputs?

**Reply by KevinMusgrave** [reply]
I'm more interested in the meaning of the suffixes, like `GetPayload` and `CreateInput`.

> Do you mind creating a [GitHub Docs Issue](https://github.com/prisma/docs/issues/new?assignees=&labels=docs&template=doc_request.md) in the prisma docs repository so that our docs team can provide their inputs?

Done, here it is: https://github.com/prisma/docs/issues/4317

**Reply by nurul3101** [reply]
Awesome, thanks for creating the Docs Issue! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
